libs/ledger-live-common/dependencies.md

```diff
@@ -47,6 +47,7 @@ yarn upgrade-interactive -i --latest
 |@solana/web3.js         | Solana coin integration        | **BLOCKED BY LLM because of BigInt in RN (ticket missing)**                               |
 |@taquito/ledger-signer  | Tezos coin integration         | **BLOCKED BY LLM (ticket missing)**                               |
 |@taquito/taquito        | Tezos coin integration         | **BLOCKED BY LLM (ticket missing)**                               |
+|@taquito/utils          | Tezos coin integration         | **BLOCKED BY LLM (ticket missing)**                               |
```

Comment:
mmh why is it blocked now tho? i think this PR unblock this specific topic 🤔

Reply:
I need to take time to test it on iOS, I remember we got some issues with iOS, but it's seems they replace libsodium with stable-lib so it should be alright

Reply:
i think we did some magic when porting to the mono-repo so we could patch the libsodium bullshit.

https://github.com/LedgerHQ/ledger-live/blob/develop/package.json#L103

Not sure if it's related but worth checking it out

Reply:
i'm not sure if my original message was clear, but we need to remove the mention `| **BLOCKED BY LLM (ticket missing)**                               |` specifically because I can see this PR now updates it so this PR literally remove the blocker 🤔  (I mean, after the verification there are no such iOS issue.)